lib/tests/streamlit/streamlit_test.py

```diff
@@ -168,6 +168,7 @@ def test_public_api(self):
                 "cache_data",
                 "cache_resource",
                 # Experimental APIs:
+                "experimental_dialog",
```

Comment:
Here it is `experimental_dialog` but in `delta_generator_test.py` its just `dialog`...

Reply:
Yeah it's a little bit confusing. The reason is that this one uses the dialog decorator which is exposed as `experimental_dialog` to the user, and the other one is the delta generator one, which is not exposed to the user but used under the hood by the decorator. I can leave a comment to make it more clear!